src/components/reports/ReportForm.test.tsx

```diff
@@ -10,47 +10,116 @@ const clearAndType = (element: Element, value: string) => {
   userEvent.type(element, value);
 };
 
-const setDummyValues = (withDates: boolean) => {
+type DropDown = {
+  field: RegExp;
+  value: string;
+};
+
+const setDummyValues = (withDates: boolean, dropDowns?: DropDown[]) => {
   const dummyValue = "Always the same";
   const form = screen.getByRole("form");
-  const textInputs = within(form).getAllByLabelText(/^((?!resultOutput|mrn|date|address|gender).)*$/i);
-  textInputs.forEach((input) => clearAndType(input, dummyValue));
+  const textInputs = within(form).getAllByLabelText(
+    /^((?!resultOutput|date|address|gender|specimen type|search|gene symbol|follow up).)*$/i,
+  );
+
+  if (!dropDowns) {
+    textInputs.forEach((input) => {
+      clearAndType(input, dummyValue);
+    });
+  } else {
+    textInputs.forEach((input) => {
+      try {
+        clearAndType(input, dummyValue);
+      } catch (e) {
+        // swallow error if this is a drop-down field
+        const result = (e as Error).message;
+        if (!result.includes("clear currently only supports input and textarea elements")) {
+          throw e;
+        }
+      }
+    });
+    dropDowns.map((dd) => {
+      const field = within(form).getByLabelText(dd.field);
+      userEvent.selectOptions(field, dd.value);
+    });
```

Comment:
yeah I think your read is right, got annoyed when we were trying to clear a select drop down. This way if the error that we get during processing is that then we don't do anything, otherwise we still throw any other errors encountered